.github/workflows/build_java_hello_world_reusable.yml

```
@@ -0,0 +1,47 @@
+name: Build and test Java example
+on:
+  workflow_call:
+    inputs:
+      cedar_policy_ref:
```

Probably makes sense to update this variable name
```suggestion
      cedar_java_ref:
```